### PR TITLE
add new device types Core200S and Dual 200S

### PR DESCRIPTION
**Pull request by webdjoe**

### CODE REVIEW

**Comment by Godfather95** [comment]
@webdjoe with #105 the test for the Dual200S ist successfull

File: src/pyvesync/vesync.py [hunk 1]
```diff
@@ -51,14 +51,17 @@
...
 }
 
 _DEVICE_TYPES_DICT: Dict[str, List[str]] = dict(
     outlets=['wifi-switch-1.3', 'ESW03-USA',
              'ESW01-EU', 'ESW15-USA', 'ESO15-TB'],
     switches=['ESWL01', 'ESWL03', 'ESWD16'],
     fans=['LV-PUR131S', 'Classic200S', 'Classic300S', 'Core200S',
-          'Core300S', 'Core400S', 'Dual200S'],
+          'Core300S', 'Core400S', 'Dual200S',
+          'LUH-D301S-WEU', 'LUH-D301S-WEU'],
```
Comment (FidgetyRat) [inline src/pyvesync/vesync.py]: Copy/Paste issue here? Missing the LAP-C201S-AUSR and duplicated the LUH-D301S-WEU device.

**Review by FidgetyRat — CHANGES_REQUESTED** [review]
I only reviewed the changes for the Core200S purifier.

File: src/pyvesync/vesync.py (hunk 1 above)
Comment (webdjoe) [reply src/pyvesync/vesync.py]: Thank you for catching that!

**Review by Godfather95 — APPROVED** [review]

**Comment by Godfather95** [comment]
@webdjoe Can you publish a new version of the package so we can use in in [https://github.com/home-assistant/core](url)

**Comment by kevchu3** [comment]
@webdjoe Seconding what @Godfather95 said, we'd like to get the latest Vesync compiled package available so that we can use it in Home Assistant

**Comment by webdjoe** [comment]
@kevchu3 @Godfather95 I will get this up tonight

**Comment by kevchu3** [comment]
@webdjoe Thanks for merging this PR.  I've done some testing and looks like pyvesync is picking up the Dual 200S humidifier as VeSyncHumid200300S :)

```
$ python3
Python 3.10.2 (main, Jan 17 2022, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyvesync import VeSync
>>> manager = VeSync("EMAIL", "PASSWORD", "US")
>>> manager.login()
True
>>> manager.update()
>>> manager.fans
[DevClass: VeSyncHumid200300S,                Name:(omitted), Device No: None,                DevStatus: off, CID: (omitted)]
```

As a next step, would you be able to publish a new release, so HA can update their code?
https://github.com/home-assistant/core/pull/62907#discussion_r783475024

**Comment by ekinnee** [comment]
Any word on a new release?

**Comment by webdjoe** [comment]
@ekinnee Sorry for delay, released!

**Pull request merged**